packages/canary-integration/src/__tests__/capability.test.ts

```diff
@@ -114,7 +115,7 @@ describe('CACAO Integration test', () => {
       const deterministicDocument = await TileDocument.deterministic(ceramic, {
         deterministic: true,
         family: 'testCapabilities1',
-        controllers: [`did:pkh:eip155:1:${wallet.address}`],
+        controller: [`did:pkh:eip155:1:${wallet.address}`],
```

Comment:
This doesn't seem right. Passing an array to the `controller` prop?

Reply:
oops, typo.  Fixed